test/files/base64.iced

```diff
@@ -19,6 +19,6 @@ exports.test_b64_compat = (T,cb) ->
     for padlen in [0...pad.length]
       buf = Buffer.concat [ pad[0...padlen], x[0...i] ]
       e1 = b64stripped.encode buf
-      e2 = b64enc.encode buf
+      e2 = b64enc.encode buf, true
```

Comment:
Ah stupid me, I thought this was an example of the `old_shift` flag. I didn't mean to say that we should change the Base64 encoder.

Your call on whether the object-destructuring style is worth doing for the shift argument.